python/tvm/meta_schedule/relay_integration.py

```diff
@@ -390,16 +353,20 @@ def compile_relay(
     from tvm import relay
 
     # pylint: enable=import-outside-toplevel
-    mod, target, params, pass_config, executor = _normalize_params(
-        mod, target, params, pass_config, executor
-    )
+    mod, target, params, executor = _normalize_params(mod, target, params, executor)
+    pass_ctx = transform.PassContext.current()
+    pass_config = dict(pass_ctx.config)
```

Comment:
To prevent breaking user's script after this change, how about setting a default ctx with opt_level = 3 if there is no current ctx?

Reply:
Right now there is no way to determine whether or not if the current pass context is the default or not. It might be possible to add though.

I is not consistent with the rest of TVM to default the opt_level to 3 in meta schedule. Everywhere else defaults to 2. This does mean that we might break some user scripts.